src/app/tl-header/tl-header.component.html

```diff
@@ -160,61 +160,37 @@
             </a>
           </wvr-dropdown-menu-item>
           <wvr-dropdown-menu-item>
-            <svg class="bi bi-arrow-right-circle" viewBox="0 0 16 16"  xmlns="http://www.w3.org/2000/svg">
-              <path fill-rule="evenodd" d="M8 15A7 7 0 1 0 8 1a7 7 0 0 0 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"/>
-              <path fill-rule="evenodd" d="M7.646 11.354a.5.5 0 0 1 0-.708L10.293 8 7.646 5.354a.5.5 0 1 1 .708-.708l3 3a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708 0z"/>
-              <path fill-rule="evenodd" d="M4.5 8a.5.5 0 0 1 .5-.5h5a.5.5 0 0 1 0 1H5a.5.5 0 0 1-.5-.5z"/>
-            </svg>
+            <tl-icon set="tl" name="arrow-right-circle"></tl-icon>
```

Comment:
This would be fine to reference from the bootstrap set.

Reply:
Tl-icon is now referencing from bootstrap folder